documentation/modules/adding-hook-using-ui.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Migration_Toolkit_for_Virtualization/master.adoc
+
+:_content-type: PROCEDURE
+[id="adding-migration-hook-via-ui_{context}"]
+= Adding a migration hook to a migration plan using the {ocp} web console
+
+You can add a migration hook to an existing migration plan using the {ocp} web console. Note that you will need to execute one command in the {project-first} CLI.
```

Comment:
```suggestion
You can add a migration hook to an existing migration plan using the {ocp} web console. Note that you will need to run one command in the {project-first} CLI.
```

i think `excute` might be a problematic word

Reply:
...and I'd remove "will"